src/db/effect_bundles_to_effects_junctions_tables_vco.tsv

```diff
@@ -79,8 +79,8 @@ vco_victory_payload_ksl_ort_1_there_can_only_be_one	wh3_main_effect_province_gro
 vco_victory_payload_ksl_ort_1_there_can_only_be_one	wh3_main_effect_ksl_motherland_cost_mod	-25.0	faction_to_faction_own_unseen	start_turn_completed
 vco_victory_payload_ksl_ort_2_the_last_bastion	wh2_dlc13_effect_force_stat_barrier_defending_siege	600.0	faction_to_force_own_unseen	start_turn_completed
 vco_victory_payload_ksl_ort_2_the_last_bastion	wh2_main_effect_force_enemy_campaign_movement_range	-20.0	faction_to_region_own	start_turn_completed
-vco_victory_payload_ksl_ort_3_wololo_1	wh2_main_effect_economy_gdp_mod_trade_sea_global	30.0	faction_to_region_own_unseen	start_turn_completed
```

Comment:
Is this also reflected in the Markdown for the faction? I guess it was right in the code, wrong in the docs, but just checking.

Reply:
The markdown still does not match the rewards you guys went with in the end. Like stuff was shuffled around, recruit flagellants (a religious zealot unit) was replaced with a templehof luminark, etc.

Reply:
Update it then too please

Reply:
It didn't let me just update it. I had to do another pull request or whatever.

I see what is wrong is that everything is the same except Flagellants was replaced with Templehof Luminark, and then everything was scrambled into a different order. Fixed now.